lib/elixir/lib/enum.ex

```diff
@@ -1653,6 +1653,9 @@ defmodule Enum do
   It assumes that the sample being returned can fit into memory;
   the input `enumerable` doesn't have to, as it is traversed just once.
 
+  If a range is passed into the function it will not be expanded so it
```

Comment:
I just also like @whatyouhide's version as it says "this function will pick" instead of "Elixir will pick". :bowtie: